### DISCUSSION TITLE
[Feature] Image File Share Function With Direct URL

### DISCUSSION THREAD

**Discussion by Majoraslayer** [immich-app/immich#7537] [Feature Request]
### The feature

It would be great if we could get a share feature that creates a direct URL to an image file.  This would be a great way to use Immich as hosting for embedding photos in forum posts, or other web pages as necessary.  It would allow users to have a convenient self-hosted alternative to Imgur and Photobucket.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3556
